pandas/tools/tests/test_merge.py

```diff
@@ -2255,6 +2255,16 @@ def test_with_mixed_tuples(self):
         df2 = DataFrame({ u'B' : 'foo', (u'B',1) : 'bar' },index=range(2))
         result = concat([df1,df2])
 
+    def test_concat_Nat_series(self):
+        # GH 11693
+        # test for merging NaT series with datetime series.
+        x = pd.Series( pd.date_range('20151124 08:00', '20151124 09:00', freq = '1h', tz = "US/Eastern") )
+        y = pd.Series( pd.date_range('20151124 10:00', '20151124 11:00', freq = '1h', tz = "US/Eastern") )
+        y[:] = pd.NaT
+        xy = pd.concat([x,y])
```

Comment:
if we have other tests in merge which test this kind of thing, pls move to a similar location (I don't know if we do)